test/test-pmctl-start-stop-restart.js

```diff
@@ -0,0 +1,53 @@
+var fs = require('fs');
+var tap = require('tap');
+var fmt = require('util').format;
+var helper = require('./helper-async');
+
+var useREST = process.env.STRONGLOOP_PM;
+delete process.env.STRONGLOOP_PM;
+
+process.env.STRONGLOOP_CLUSTER = 0;
+
+function test(name, callback) {
```

Comment:
I like it in principle, would lie to avoid this duplicate `test()` function, though.